src/call/mod.rs

```diff
@@ -14,9 +14,10 @@
 pub mod client;
 pub mod server;
 
+use std::cmp;
 use std::io::{self, BufRead, ErrorKind, Read};
 use std::sync::Arc;
-use std::{cmp, mem, ptr, slice, usize};
+use std::{mem, ptr, slice, usize};
```

Comment:
Why?

Reply:
GitHub is only showing 4 lines which makes my comment misleading.

Look at this pic:
![image](https://user-images.githubusercontent.com/16398479/47398644-fa0f1900-d702-11e8-99b8-705739c10eaa.png)